src/node.js

```diff
@@ -178,12 +178,27 @@
   }
 
   startup.setupProcessObject = function() {
-    process._setupProcessObject(setPropByIndex);
+    const _hrtime = process.hrtime;
+    const hrValues = new Uint32Array(3);
```

Comment:
Maybe add a short comment here or in hrtime() explaining what the fields contain after an update.